support/azure/virtual-machines/linux/troubleshoot-linux-rhui-certificate-issues.md

```diff
@@ -110,7 +110,7 @@ All the commands in the following steps should be run by using root privileges o
 3. Reinstall the corresponding `rhui-azure` package by running the `yum reinstall` command:
 
    ```bash
-   sudo yum reinstall $(rpm -qa | grep -i rhui-azure) --disablerepo=* --enablerepo="*microsoft-azure*"
+   sudo yum reinstall $(rpm -qa | grep -i rhui-azure) --disablerepo=* --enablerepo="*microsoft*"
```

Comment:
@AmandaAZ please keep this line using microsoft-azure as well, as the PR creator proposed. That's OK

Reply:
OK. I will create a new PR to update it.